app/scripts/controllers/membership.js

```diff
@@ -45,7 +45,8 @@ angular
         update: {
           subject: {
             success: _.template('The role "<%= roleName %>" was given to "<%= subjectName %>".'),
-            error: _.template('The role "<%= roleName %>" was not given to "<%= subjectName %>".')
+            error: _.template('The role "<%= roleName %>" was not given to "<%= subjectName %>".'),
```

Comment:
not new, but could you change "was not" to "could not be" ?

I'd probably change "given" to "granted" on all of these.  You generally talk about granting someone access to something.  Other opinions  @spadgett @rhamilto  ?